packages/shared/src/components/cards/SourceButton.tsx

```diff
@@ -16,14 +15,15 @@ interface SourceButtonProps {
 
 export default function SourceButton({
   source,
-  link,
```

Comment:
This was introduced for the notification avatar, It should always prefer a direct link (`targetURL`) so think we should keep it.